update-tags.mjs

```diff
@@ -0,0 +1,65 @@
+import { readFile, writeFile } from 'fs/promises'
+import { execSync } from 'child_process'
+import { glob as baseGlob } from 'glob'
+
+const version = JSON.parse(await readFile('package.json')).version
+if (!/[0-9]+\.[0-9]+\.[0-9]+.*/.test(version)) {
```

Comment:
Why do we need to validate the version number? Why not just take its value verbatim, whatever it is?

Reply:
Yeah, just remove it. The version number `0.1` would fail validation, for example.